analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -91,7 +92,7 @@ class ClearlyDefinedPackageCurationProvider(server: Server = Server.PRODUCTION)
         val curation = try {
             // TODO: Maybe make PackageCurationProvider.getCurationsFor() a suspend function; then all derived
             //       classes could deal with coroutines more easily.
-            runBlocking { service.getCuration(type, provider, namespace, pkgId.name, pkgId.version) }
+            runBlocking(Dispatchers.IO) { service.getCuration(type, provider, namespace, pkgId.name, pkgId.version) }
```

Comment:
Should the commit message title actually say: "...Avoid blocking the main thread..."?

Reply:
That's what I was meaning to express, yes, but maybe I phrased it badly. Changed.